tests-e2e/ocw-ci-test-course/video-tabs.spec.ts

```diff
@@ -2,6 +2,26 @@ import { test, expect } from "@playwright/test"
 import { CoursePage } from "../util"
 import { VideoElement } from "../util/VideoElement"
 
+test("Verify that the Download Button works for multiple embed videos in a page", async ({
```

Comment:
```suggestion
test("that the Download Button works for multiple embed videos in a page", async ({
```
"verify" just means "test" anyway.

Reply:
Done